Core/Graphics/Source/Graphics.cpp

```diff
@@ -25,6 +30,13 @@ namespace Babylon
         return graphics;
     }
 
+    std::unique_ptr<Graphics> Graphics::CreateGraphics(const ContextConfiguration& config)
+    {
+        std::unique_ptr<Graphics> graphics{ new Graphics() };
```

Comment:
nit: formatting (in a few places)

```suggestion
        std::unique_ptr<Graphics> graphics{new Graphics()};
```